contrib/how-to-build.md

```diff
@@ -0,0 +1,56 @@
+# How To Build With `cmake`
+
+To build this project you can still use `npm install`, but here some advice to improve your development experience:
+
+
+## Getting Started
+
+First clone the project:
+```sh
+git clone git@github.com:realm/realm-js.git
+
+# The only branch supporting cmake at the moment is develop
+git checkout develop
+```
+
+Pull the submodules:
+
+```sh
+git submodule update --init --recursive
+```
+> If this folder `src/object-store` exist then do a `git clean -xfd`.
+
+
+
+## Configuration
+Now you should get [cmake-js](https://www.npmjs.com/package/cmake-js) which is a [node-gyp](https://github.com/nodejs/node-gyp) replacement:
```

Comment:
`cmake-js` is a dev dependency so `npm install` should install it.

Of course, if you want to run `cmake-js` manually, it might be easier to install it globally.